src/mappings/utils.ts

```diff
@@ -2,6 +2,10 @@ import {CosmosBlock, CosmosEvent, CosmosMessage, CosmosTransaction} from "@subql
 import {Account, Interface, UnprocessedEntity} from "../types";
 import {createHash} from "crypto";
 import {Attribute} from "@cosmjs/stargate/build/logs";
+import {LegacyBridgeSwapStructure} from "./wasm/contracts/bridge";
+import {CW20Structure} from "./wasm/contracts/cw20";
```

Comment:
This was supposed to be in https://github.com/fetchai/ledger-subquery/commit/d7c166fd6f3d560e1801a581bab2cfa231faf22c